### PR TITLE
Add missing BTFs for running kitchen tests locally

### PR DESCRIPTION
**Pull request by ISauve**
### What does this PR do?

Adds copying (and minimizing if possible) of BTFs to the kitchen prepare invoke task.

### Describe how to test/QA your changes

Run kitchen tests locally and verify they pass.

### Reviewer's Checklist

- [x] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [x] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [x] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [x] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: tasks/system_probe.py [hunk 1]
```diff
@@ -1143,6 +1149,52 @@ def tempdir():
...
+            ctx,
+            source_dir=f"{btf_dir}/btfs-{arch}",
+            output_dir=f"{btf_dir}/minimized-btfs",
+            input_bpf_programs=co_re_programs
+        )
+
+        ctx.run(f"cd {btf_dir}/minimized-btfs && " +
+                f"{sudo} tar -cJf minimized-btfs.tar.xz * && " +
+                f"{sudo} mv minimized-btfs.tar.xz {files_dir}")
+    except:
```
Comment (brycekahle) [inline tasks/system_probe.py:1173]: This is too wide of a net, and is catching other errors besides the `bpftool` version. I'd suggest calling `check_for_bpftool_compatability(ctx)` in a `try/except` block, on its own.

**Review by KSerrania — APPROVED** [review]

**Review by KSerrania — COMMENTED** [review]
LGTM for Agent Platform-related files, but the Python linter is complaining and eBPF tests are failing.

File: tasks/system_probe.py [hunk 2]
```diff
@@ -644,6 +644,8 @@ def kitchen_prepare(ctx, windows=is_windows, kernel_release=None):
         if os.path.exists(cf):
             shutil.copy(cf, files_dir)
 
+    kitchen_prepare_btfs(ctx, files_dir)
```
Comment (brycekahle) [inline tasks/system_probe.py:647]: This is called in the CI too, not just for local kitchen testing

File: tasks/system_probe.py [hunk 3]
```diff
@@ -1147,6 +1157,45 @@ def tempdir():
...
+def kitchen_prepare_btfs(ctx, files_dir, arch=CURRENT_ARCH):
+    btf_dir = "/opt/datadog-agent/embedded/share/system-probe/ebpf/co-re/btf"
+
+    if arch == "x64":
+        arch = "x86_64"
+    elif arch == "arm64":
+        arch = "aarch64"
+
+    if not os.path.exists(f"{btf_dir}/kitchen-btfs-{arch}.tar.xz"):
+        exit("BTFs for kitchen test environments not found. Please update & re-provision your dev VM.")
```
Comment (brycekahle) [inline tasks/system_probe.py:1169]: This is what is currently causing the ebpf jobs to fail

**Review by brycekahle — APPROVED** [review]

**Review by iglendd — APPROVED** [review]

**Pull request merged**